decidim-forms/app/views/decidim/forms/questionnaires/show.html.erb

```diff
@@ -3,154 +3,40 @@
-                            <%= decidim_sanitize_editor translated_attribute questionnaire.tos %>
-                          </div>
-                        </div>
-                      <% end %>
+  <%= render partial: "decidim/shared/component_announcement" if current_component.manifest_name == "surveys" %>
 
-                      <%= cell(
-                        "decidim/forms/step_navigation",
-                        step_index,
-                        total_steps: @form.total_steps,
-                        button_disabled: !current_participatory_space.can_participate?(current_user),
-                        form: form
-                      ) %>
-                    </div>
-                  <% end %>
-                <% end %>
-              </div>
-            <% end %>
-          <% else %>
-            <div class="answer-questionnaire">
-              <h3 class="section-heading"><%= t(".answer_questionnaire.title") %></h3>
-              <p>
-                <%= t(".answer_questionnaire.anonymous_user_message", sign_in_link: decidim.new_user_session_path, sign_up_link: decidim.new_user_registration_path).html_safe %>
-              </p>
+  <h1 class="h1 decorator my-12"><%= translated_attribute questionnaire.title %></h1>
 
-              <ol>
-                <%= cell("decidim/forms/question_readonly", collection: @questionnaire.questions.not_conditioned) %>
-              </ol>
-            </div>
-          <% end %>
-        <% else %>
-          <div class="section">
-            <div class="callout warning">
-              <h3 class="heading4"><%= t(".questionnaire_closed.title") %></h3>
-              <p><%= t(".questionnaire_closed.body") %></p>
-            </div>
-          </div>
-        <% end %>
-      </div>
-    </div>
+  <div class="text-xl text-gray-2 mb-10">
+    <%= decidim_sanitize_editor translated_attribute questionnaire.description %>
   </div>
-</div>
 
-<% content_for :js_content do %>
-  <%= javascript_pack_tag "decidim_forms" %>
+  <% unless questionnaire_for.try(:component)&.try(:published?) %>
+    <%= cell("decidim/announcement", t("decidim.forms.questionnaires.show.questionnaire_not_published.body"), callout_class: "warning") %>
+  <% end %>
+
+  <% if allow_answers? %>
+    <% if visitor_can_answer? %>
+      <% if visitor_already_answered? %>
+        <%= cell("decidim/announcement", { title: t("decidim.forms.questionnaires.show.questionnaire_answered.title"), body: t("decidim.forms.questionnaires.show.questionnaire_answered.body") }) %>
+      <% else %>
+        <%= render partial: "decidim/forms/questionnaires/questionnaire" %>
+      <% end %>
+    <% else %>
+      <%= render partial: "decidim/forms/questionnaires/questionnaire_readonly" %>
+    <% end %>
+  <% else %>
+    <%= cell("decidim/announcement", { title: t("decidim.forms.questionnaires.show.questionnaire_closed.title"), body: t("decidim.forms.questionnaires.show.questionnaire_closed.body") }, callout_class: "warning") %>
```

Comment:
I believe the same margin issue also applies to this announcement as above, so please check this one too.